src/frontend/src/components/TestButton/TestButton.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { ReactElement } from "react";
+import classes from "./styles.module.css";
+
+export default function TestButton(): ReactElement {
```

Comment:
```suggestion
export default function TestButton() {
```

IMO there is no need to type the return of a react component.

Reply:
The return type is inferred by typescript. 
If you absolutely want to use typing with a component, I suggest you to type the function itself using `React.FC` this way:

```
const TestButton: React.FC = () => <button className={classes.root}>Test</button>;

export default TestButton
```

Reply:
And here an article you can read about typing a react component: https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript